resources/views/bread/edit-add.blade.php

```diff
@@ -83,9 +83,31 @@ class="form-edit-add"
 
                         <div class="panel-footer">
                             <button type="submit" class="btn btn-primary save">{{ __('voyager::generic.save') }}</button>
+                            <button type="button" class="btn btn-danger js-delete">{{ __('voyager::generic.delete') }}</button>
                         </div>
                     </form>
 
+                    {{-- Single delete modal --}}
+                    <div class="modal modal-danger fade" tabindex="-1" id="delete_modal" role="dialog">
```

Comment:
Why do you have a "delete" modal here, when there's one just a few lines below this? How are they different?

Reply:
The difference is that the previously existing modal is only for deleting (multiple) media/images.
I didn't want to change too much there, as I don't know the inner workings of that flow.